core/model/modx/transport/modtransportpackage.class.php

```diff
@@ -432,7 +455,7 @@ public function transferPackage($sourceFile, $targetDir) {
                     $transferred= $cacheManager->writeFile($target, $content);
                 }
             } else {
-                $this->xpdo->log(xPDO::LOG_LEVEL_ERROR,'MODX could not download the file. You must enable allow_url_fopen, cURL or fsockopen to use remote transport packaging.');
+                $this->xpdo->log(xPDO::LOG_LEVEL_ERROR, 'MODX could not download the file. ' . (isset($status) ? 'Server responded with status code ' . $status . '.' : 'You must enable allow_url_fopen, cURL or fsockopen to use remote transport packaging.'));
```

Comment:
To maintain readability it might be better to use `sprintf`.

Reply:
Can you help me out with that?